modules/targetanalysis/src/target_loader.cpp

```diff
@@ -66,61 +73,63 @@ using namespace std;
 * }
 */
 
-/*
-class targetLoader{
-      targetLoader(string f){
-             file = f;     
-      }
-      public:
-             TargetParameters getParameters();
-             class TargetParameters{
-                   public:
-                          string* parametersTypes;
-                          Parameter* parameters;
-                          unsigned int parameterCount;
-                          class Parameter{
-                                public:
-                                       bool enabled;
-                                       double value;
-                                       unsigned int confidence;
-                          }
-             }
-      private:
-              string file;
-              readFile(string fileLocation);
-              
-}
-
-
-
 
+/*
 TargetParameters targetLoader:getParameters(){
        return 0;
 }
-
-char* targetLoader:readFile(string fileLocation){
+*/
+bool TargetLoader::readFile(string fileLocation, char* data){
      ifstream jFile;
-     char* memblock;
      
      jFile.open(fileLocation, ios::binary | ios::ate);
```

Comment:
The reason this was failing to compile without c++11 is that c++11 added ifstream::open(string, openmode), while prior versions just had ifstream::open(const char*, openmode)